src/icastats.c

```diff
@@ -10,7 +10,7 @@
  *             Benedikt Klotz <benedikt.klotz@de.ibm.com>
  *             Ingo Tuchscherer <ingo.tuchscherer@de.ibm.com>
  *
- * Copyright IBM Corp. 2009, 2010, 2011, 2014
+ * Copyright IBM Corp. 2009, 2010, 2011, 2014, 2019
```

Comment:
Hmm, that's not a correct copyright date specification. You have only first and last year.

Reply:
I would say we leave the list of years in the comment, but fix the string in the COPYRIGHT constant.

Reply:
While you are at it please also change it here.

Reply:
ok, done.